frontend/src/scenes/insights/EditorFilters/YAxisScaleSettings.tsx

```diff
@@ -0,0 +1,21 @@
+import { LemonCheckbox } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { insightLogic } from 'scenes/insights/insightLogic'
+
+import { insightVizDataLogic } from '../insightVizDataLogic'
+
+export function YAxisScaleSettings(): JSX.Element | null {
+    const { insightProps } = useValues(insightLogic)
+    const { yAxisScaleType } = useValues(insightVizDataLogic(insightProps))
+    const { updateInsightFilter } = useActions(insightVizDataLogic(insightProps))
+
+    return (
+        <LemonCheckbox
```

Comment:
fly-by that I think this should be a select "linear" or "log10" rather than a checkbox

but I'll defer to @PostHog/team-product-analytics

Reply:
:thinking: :shrug: 

Less clicks - better, isn't it? I don't mind either option though. Thanks for routing:)

Reply:
I agree with Paul that this should be a "Linear"/"Log 10" select!

Reply:
Turned it into a segmented button – a perfect use for it, given that we have exactly two options here. Both should be shown explicitly, which is why a checkbox isn't ideal – but a select would be overkill.

<img width="230" alt="Screenshot 2024-06-19 at 15 26 18" src="https://github.com/PostHog/posthog/assets/4550621/106c2e66-bf97-4471-812f-815427e4debd">

Reply:
Oh, thanks!